server/apps/acсount/views.py

```diff
@@ -0,0 +1,49 @@
+from .models import User, Team
+from .serializers import (
+    UserSerializer,
+    UserDetailSerializer,
+    UserUpdateSerializer
+    )
+
+from rest_framework import generics
+from rest_framework.permissions import IsAuthenticated
+
+
+class UserListView(generics.ListAPIView):
+    serializer_class = UserSerializer
+    permission_classes = (IsAuthenticated,)
+
+    def get_queryset(self):
+        queryset = User.objects.all()
```

Comment:
does it make good practice if instead .all() to use .only(ser. field)?

Reply:
U can ask about it in the chatgpt :smile: 

In my practice, I have always used a simple `.all()` construction and filtered instances using serializer fields.

I think that in development, using `.all()` can help solve some problems.
But in production mode, it is better to use `.only(...fields)` to extract only the necessary fields from the database (this way the size of the database response will be smaller, which means that the request execution time will decrease).